environment/CMakeLists.txt

```diff
@@ -27,6 +27,35 @@ include_directories(${CMAKE_SOURCE_DIR}/zstd-1.3.0/lib/legacy)
 
 get_property(dirs DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR} PROPERTY INCLUDE_DIRECTORIES)
 
+# Use git describe as the version for the executable, based on code from
+# http://eatmyrandom.blogspot.com/2010/06/automate-version-numbering-using-git.html
+find_package(Git)
+if( NOT GIT_FOUND AND EXISTS "${CMAKE_SOURCE_DIR}/../.git" )
```

Comment:
Should this be `GIT_FOUND`, not `NOT GIT_FOUND`?

Reply:
Yep, sorry I just noticed that I left that in from testing the else case.